lib/ruy/outcome.rb

```diff
@@ -1,24 +1,39 @@
 module Ruy
+  # Represents a possible result (outcome) of evaluating a rule set
+  #
+  # Additionally, an outcome can be composed of rules. In that case, the outcome will evaluate to
+  # its value if all of them are evaluated successfully.
```

Comment:
I think the indentation rule applies to attributes. 

Here's a description sample from YARD codebase: https://github.com/lsegal/yard/blob/master/lib/yard/parser/source_parser.rb#L255